### ISSUE TITLE
Sprint 1 closeout tasks:

### ISSUE THREAD

**Issue by bieryAtFnal** [DUNE-DAQ/ddpdemo#17]
* create and/or advertise a Wiki page that describes how anyone can create a SW environment, clone the _ddpdemo_ repo, build and run the DataWriterDemo.
* delete the branches that we used for development (that have now been successfully merged to _develop_)
* close the GitHub Issues associated with the Sprint
* tag the code

**Comment by bieryAtFnal** [comment]
[DDPDemo_Sprint1_RetrospectiveNotes_KAB_23Sep2020.pdf](https://github.com/DUNE-DAQ/ddpdemo/files/5268694/DDPDemo_Sprint1_RetrospectiveNotes_KAB_23Sep2020.pdf)

I'm adding a copy of our notes from the Sprint 1 Retrospective meeting here (simply as a convenience since it seems like a reasonable place to store them)

**Comment by bieryAtFnal** [comment]
I've created a Wiki page that describes how anyone can build the code that is part of Sprint 1 and test it.  That page is [here](https://github.com/DUNE-DAQ/ddpdemo/wiki/Validating-the-deliverables-from-Sprint-1).

**Comment by bieryAtFnal** [comment]
I've started closing Issues that were part of Sprint 1.

A few notes on this:

- to help me/us keep track of which Issues were part of Sprint 1, I created a GitHub Milestone with that name and associated all of the relevant Issues with that Milestone.
- I wondered whether closing an Issue would remove it from the Scrum Board (project), but it didn't. So, I've started 'archiving' the "cards" that were part of Sprint 1 and are Done, and the associated Issues have been closed. We can go back and look at these in the Scrum Board Project archive later, if we want.
- Another way that we could handle different Sprints, I imagine, is to create a new Project (Scrum Board) for each Sprint.  I like the ability to put cards on the Scrum Board that are not yet part of a Sprint, though, so it seems like we want the Scrum Board to be above individual Sprints.
